packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -77,6 +80,21 @@ export class WaferMap<
     @attr({ attribute: 'color-scale-mode' })
     public colorScaleMode: WaferMapColorScaleMode = WaferMapColorScaleMode.linear;
 
+    /**
+     * @internal
+     */
+    public isWorkerAlive = false;
```

Comment:
Why is this state being kept here, when the only thing that cares about it is the `WorkerRenderer`?